### PR TITLE
Additional restructuring suggestions for CAP Java Remote Services section

### PR DESCRIPTION
**Pull request by beckermarc**

### CODE REVIEW

File: java/cqn-services/remote-services.md [hunk 1]
```diff
@@ -360,57 +348,61 @@ This is especially useful when integrating micro-services, which may have differ
...
             BizPartnerOAuth2PropertySupplier::new);
 }
 ```
 
 [Learn more about registering OAuth2PropertySupplier in the **SAP Cloud SDK documentation**.](https://sap.github.io/cloud-sdk/docs/java/features/connectivity/service-bindings#customization){.learn-more}
 
-### Register Destinations
+### Using Destinations { #native-destinations }
+
+If the URL and credentials of the remote API is configured are available as a destination in SAP BTP Destination Service, you can use Cloud SDK's `DestinationAccessor` API to load the destination based on its name. In a second step, `HttpClientAccessor` is used to create an instance of `HttpClient`:
```
Comment (StefanHenke) [inline java/cqn-services/remote-services.md:389]:
```suggestion
If the URL and credentials of the remote API is configured as a destination in SAP BTP Destination Service, you can use Cloud SDK's `DestinationAccessor` API to load the destination based on its name. In a second step, `HttpClientAccessor` is used to create an instance of `HttpClient`:
```

**Review by StefanHenke — APPROVED** [review]

File: java/cqn-services/remote-services.md [hunk 2]
```diff
@@ -72,143 +74,129 @@ To learn more about this, have a look at the section [Importing Service Definiti
...
-In contrast to BTP destinations in general, it can be created and refreshed as part of the application lifecycle, that is, application deployment.
-Hence, the location and security aspects of remote services are transparent to CAP applications in the case of service bindings.
+[Learn more about registering OAuth2PropertySupplier in the **SAP Cloud SDK documentation**.](https://sap.github.io/cloud-sdk/docs/java/features/connectivity/service-bindings#customization){.learn-more}
 
-#### Binding to Local Service
+#### Binding to a Service with shared Identity
 
-In the following example, the remote API is running as another CAP application within the same SaaS application. Both, the calling CAP application and the remote API, are bound to the same (shared) XSUAA service instance and accept JWT tokens issued by the single XSUAA instance.
+In case the remote API is available within the same SaaS application and using the same (shared) XSUAA service instance for authentication no service broker-based reuse service is required.
```
Comment (renejeglinsky) [inline java/cqn-services/remote-services.md:142]:
```suggestion
If the remote API is available within the same SaaS application and using the same (shared) XSUAA service instance for authentication, no service broker-based reuse service is required.
```

File: java/cqn-services/remote-services.md [hunk 3]
```diff
@@ -72,143 +74,129 @@ To learn more about this, have a look at the section [Importing Service Definiti
...
+The class `SomeReuseServiceOAuth2PropertySupplier` needs to be provided by you extending the Cloud SDK base class `DefaultOAuth2PropertySupplier`.
 
-If the remote API is running on SAP BTP, it's likely that you can leverage Service Binding-based _Remote Services_. 
-The CAP Java SDK extracts the relevant information from the service binding to connect to the remote API. The advantage of service-binding-based _Remote Services_ is the simpler usage. 
-A service binding abstracts from several aspects of remote service communication. For instance, it provides authentication information and the URL of the service. 
-In contrast to BTP destinations in general, it can be created and refreshed as part of the application lifecycle, that is, application deployment.
-Hence, the location and security aspects of remote services are transparent to CAP applications in the case of service bindings.
+[Learn more about registering OAuth2PropertySupplier in the **SAP Cloud SDK documentation**.](https://sap.github.io/cloud-sdk/docs/java/features/connectivity/service-bindings#customization){.learn-more}
```
Comment (renejeglinsky) [inline java/cqn-services/remote-services.md:138]: Can we put that in a details section? Seems to be an edge case/troubleshooting, right?

Comment (renejeglinsky) [reply java/cqn-services/remote-services.md:138]: Expanded:
image

Collapsed:
image

Comment (beckermarc) [reply java/cqn-services/remote-services.md:138]: +1

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**